gcloud/_apitools/http_wrapper.py

```diff
@@ -91,8 +90,7 @@ def _Httplib2Debuglevel(http_request, level, http=None):
     httplib2.debuglevel = old_level
     if http is not None:
         for connection_key, old_level in http_levels.items():
-            if connection_key in http.connections:
-                http.connections[connection_key].set_debuglevel(old_level)
+            http.connections[connection_key].set_debuglevel(old_level)
```

Comment:
There are no users outside this module:  the (private) `_Httplib2DebugLevel` generator is a `contextlib.contextmanager` implementation, used only by `_MakeRequestNoRetry` to wrap around `http.request`, which pretty much has to be `httplib2.Http.request` (given the oddball signature).  That method *adds* keys to `self.connections`, but never removes them, which makes the check unnecessary.